warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/error/ErrorShardedDataTypeHandler.java

```diff
@@ -125,7 +121,6 @@
     
     @Override
     public void setup(TaskAttemptContext context) {
-        markingFunctions = MarkingFunctionsFactory.createMarkingFunctions();
```

Comment:
Might want to investigate where this was originally added. I could be wrong, but I feel like it might have been put there for the side effects.

Reply:
Yep, `ShardedDataTypeHandler.setup` instantiates markingFunctions, but slightly differently. It uses `MarkingFunctions.Factory` rather than `MarkingFunctionsFactory`, which is compatible for webserver usage. I can revert this line. Spotbugs should still be happy b/c field masking was the root issue, not this particular assignment.

But since `ErrorShardedDataTypeHandler` is ostensibly compatible with both webserver use and with normal ingest deployment, that should mean that `MarkingFunctionsFactory` should be as well, right? In other words, what if we just changed `ShardedDataTypeHandler` to use `MarkingFunctionsFactory` instead? And leave this code as-is?

The main diff between the two `createMarkingFunctions` methods is that they load slightly different Spring config files. Seems like that could be refactored somehow, so that all the work is delegated to the `MarkingFunctions.Factory` version, as long as we make that version smart enough to overridde Spring config locations

Myeah, probably at task for a different ticket. lol

Reply:
I'll test the both ingest and web deployment for this a little more thoroughly before I update this PR. Need to make sure the field masking isn't required to keep everything working. Kinda worried that it might be

Reply:
Ingest deploy/test works fine. Was originally thinking ErrorShardedDataTypeHandler might get used or subclassed somewhere in a web service context, e.g., for QueryMetrics ingest, etc, but that does not appear to be the case. So, I think this is g2g, as-is